.config/forest.dic

```diff
@@ -30,3 +30,7 @@ tipset/SM
 tipsetkey/S
 tuple
 validator/S
+serializing
+SignedMessage
```

Comment:
This looks like code. If you mark it as code using back ticks, spellcheck will ignore it.